CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 # cmake_policy(VERSION 3.19)
 
 find_package(cetmodules 3.20.00 REQUIRED)
-project(icaruscode VERSION 09.63.00.02 LANGUAGES CXX)
+project(icaruscode VERSION 09.64.01 LANGUAGES CXX)
```

Comment:
This needs to be left out of the pull request (pull requests normally must not change the package versions with respect to the branch they will be merged into).